kernel/task.h

```diff
@@ -36,6 +37,9 @@ struct task {
     struct fdtable *files;
     struct fs_info *fs;
 
+    // Currently active AIO contexts. Contains internal lock.
+    struct aioctx_table *aioctx;
```

Comment:
This seems like it would make more sense as an inline struct instead of a pointer to a malloced block, since it always exists for exactly as long as the task.